### ISSUE TITLE
Very slow with 750 leds

### ISSUE THREAD

**Issue by penfold42** [redPanther/hypersim#5]
Is there some tk optimisation that would help ?

**Comment by redPanther** [comment]
Yes, use some Kind of canvas, instead of Tk elements. Then draw the canvas with double buffering to avoid flicker. Or make it cool and use opengl :)
